libioc/errors.py

```diff
@@ -393,7 +393,8 @@ def __init__(
     ) -> None:
         msg = (
             f"Invalid jail name '{name}': "
-            "Names may only contain alphanumeric characters and dash"
+            "Names may only contain alphanumeric characters and may not begin "
+            "with special characters ! ^ - _ ( ) [ ] { } < > , ."
```

Comment:
it's unclear from this message, whether or not the name is allowed to *contain* the these characters.

Reply:
The error message is clarified now. Thanks for spotting it!